util-images/network/netperfbenchmark/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM golang:1.15.2-alpine3.12 AS build-env
+ARG gopkg=k8s.io/perf-tests/util-images/phases/netperfbenchmark
```

Comment:
what is "phases" here and below?

Reply:
Initial implementation was using phases and this is coming from there, will update appropriately.